workspaces/src/rpc/client.rs

```diff
@@ -366,9 +378,11 @@ impl Client {
         result
     }
 
-    pub(crate) async fn wait_for_rpc(&self) -> anyhow::Result<()> {
+    pub(crate) async fn wait_for_rpc(&self) -> Result<(), RpcError> {
         let timeout_secs = match std::env::var("NEAR_RPC_TIMEOUT_SECS") {
-            Ok(secs) => secs.parse::<usize>()?,
+            // hard fail on not being able to parse the env var, since this isn't something
+            // the user should handle with the library.
+            Ok(secs) => secs.parse::<usize>().unwrap(),
```

Comment:
Shouldn't we at least supply an error message here? It would be confusing to see workspaces panic with something like "ABC is not a number"